testbed/tests/test_window.py

```diff
@@ -490,10 +490,13 @@ async def test_as_image(main_window, main_window_probe):
 
 # Test the `origin`, `position` and `screen_position`.
 async def test_screen(main_window, main_window_probe):
+    """A window can be moved to a different screen
+    and can be moved using both the absolute position
+    and relative screen position"""
```

Comment:
Again, start-of-test comments should be a single line.

Regardless, the column at which you wrap a comment is column 89, not 55.

Reply:
Changed it